openapi/p4pa-debt-position.openapi.yaml

```
@@ -29,6 +29,32 @@ paths:
           description: Invalid request
         "500":
           description: Internal server error
+  /sync/status/{debtPositionId}:
```

/<ENTITY_REF>/<OPERATION_ON_ENTITY>
```suggestion
  /debt-posistions/{debtPositionId}/finalize-sync-status:
```